integreat_cms/google_translate_api/apps.py

```diff
@@ -0,0 +1,118 @@
+"""
+Configuration of Google Translate API app
+"""
+
+from __future__ import annotations
+
+import logging
+import os
+import sys
+from typing import TYPE_CHECKING
+
+from django.apps import AppConfig
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from google.auth.exceptions import DefaultCredentialsError
+from google.cloud import translate_v2, translate_v3  # type: ignore[attr-defined]
+from google.oauth2 import service_account
+
+if TYPE_CHECKING:
+    from typing import Final
+
+    from django.utils.functional import Promise
+
+logger = logging.getLogger(__name__)
+
+
+class GoogleTranslateApiClientConfig(AppConfig):
+    """
+    Google Translate API config inheriting the django AppConfig
+    """
+
+    #: Full Python path to the application
+    name: Final[str] = "integreat_cms.google_translate_api"
+    #: Human-readable name for the application
+    verbose_name: Final[Promise] = _("Google Translate API")
+    #: The supported source languages
+    supported_source_languages: list[str] = []
+    #: The supported target languages
+    supported_target_languages: list[str] = []
+
+    # pylint: disable=too-many-nested-blocks
+    def ready(self) -> None:
+        """
+        Checking if API is available
+        """
+        # Only check availability if running a server
+        if "runserver" in sys.argv or "APACHE_PID_FILE" in os.environ:
+            if settings.GOOGLE_TRANSLATE_ENABLED:
+                try:
+                    credentials = service_account.Credentials.from_service_account_file(
+                        settings.GOOGLE_APPLICATION_CREDENTIALS
+                    )
+                except (DefaultCredentialsError, AssertionError) as e:
```

Comment:
We can probably create a separate task (or fix it now, if you like) to handle errors more carefully.
For example, if the credentials file is not found, we get FileNotFoundError and the CMS fails to start.

Reply:
I used for now the general `Eception` to catch various errors. We can make it more detailed later if that is needed.